src/components/navbar/Navbar.jsx

```diff
@@ -20,6 +22,19 @@ class Navbar extends Component {
     )
   }
 
+  renderLogout() {
+    const isLogoutRequired = history.location.pathname !== '/login' && history.location.pathname !== '/register'
+    return isLogoutRequired ? (
+      <Link to={routes.logout} className="navbarLogout">
+        <Button icon="logout">
+          Log out
+        </Button>
+      </Link>
+    ) : (
+      null
```

Comment:
Tu pourrais pas mettre le button login à la place de null ?
Même si indirectement on arrive directement sur la page login ...

Reply:
If we are in the case of `null`, it will be strange to display a login button and have a login form in the same page.